src/ResourceManager/Sql/Commands.Sql/help/AzureRM.Sql.md

```diff
@@ -407,6 +407,9 @@ Cancels the asynchronous updates operation on the database.
 ### [Stop-AzureRmSqlDatabaseExecuteIndexRecommendation](Stop-AzureRmSqlDatabaseExecuteIndexRecommendation.md)
 Stops the workflow that runs a recommended index operation.
 
+### [Stop-AzureRmSqlElasticPoolActivity](Stop-AzureRmSqlElasticPoolActivity.md)
+{{Fill in the Synopsis}}
```

Comment:
@payiAzure please fill in this synopsis

Reply:
@cormacpayne add synopsis in the AzureRmSql.md for the new command.